Source/PropertyTools.Wpf/PropertyGrid/PropertyGridOperator.cs

```diff
@@ -7,6 +7,8 @@
 // </summary>
 // --------------------------------------------------------------------------------------------------------------------
 
+using System.Windows.Controls;
```

Comment:
please following the common coding style in this project https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1200.md